ecosystem/sep-0009.md

```diff
@@ -6,120 +6,151 @@ Title: Standard KYC Fields
+| `birth_country_code`          | string | ISO Code of country of birth                                                                |
+| `bank_account_number`         | string | Number identifying bank account                                                             |
+| `tax_id`                      | string | Tax identifier of user in their country (social security number in US)                      |
+| `tax_id_name`                 | string | Name of the tax ID (`SSN` or `ITIN` in the US)                                              |
+| `occupation`                  | number | Occupation ISCO code                                                                        |
+| `employer_name`               | string | Name of employer                                                                            |
+| `employer_address`            | string | Address of employer                                                                         |
+| `language_code`               | string | primary language                                                                            |
+| `id_type`                     | string | `passport`, `drivers_license`, `id_card`, etc...                                            |
+| `id_country_code`             | string | country issuing passport or photo ID as ISO 3166-1 alpha-3 code                             |
+| `id_issue_date`               | date   | ID issue date                                                                               |
+| `id_expiration_date`          | date   | ID expiration date                                                                          |
+| `id_number`                   | string | Passport or ID number                                                                       |
+| `photo_id_front`              | binary | Image of front of user's photo ID or passport                                               |
+| `photo_id_back`               | binary | Image of back of user's photo ID or passport                                                |
+| `notary_approval_of_photo_id` | binary | Image of notary's approval of photo ID or passport                                          |
+| `ip_address`                  | string | IP address of customer's computer                                                           |
+| `photo_proof_residence`       | binary | Image of a utility bill, bank statement or similar with the user's name and address         |
+| `sex`                         | string | `male`, `female`, or `other`                                                                |
+| `proof_of_income`             | binary | Image of user's proof of income document                                                    |
+| `proof_of_liveness`           | binary | video or image file of user as a liveness proof                                             |
+
+## Shared Fields
```

Comment:
As organization fields are not widely used, maybe we should rename this header to "Financial Information Fields" or something like this?
And add in description that they could also be used for organizations too?
(I.e. we should emphasize that this fields are primarily used for financial information of a person)